dmutils/documents.py

```diff
@@ -190,6 +190,16 @@ def get_agreement_document_path(framework_slug, supplier_id, supplier_name, docu
     )
 
 
+def get_countersigned_agreement_document_path(framework_slug,
+                                              supplier_id,
+                                              document_name="countersigned-framework-agreement.pdf"):
```

Comment:
Although, something that should be parameterised is the extension as signed framework agreements can be returned as JPEGs.